server.js

```diff
@@ -2,35 +2,159 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import topMusicData from './data/top-music.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
+const Track = mongoose.model('Track', {
+  id: {
+    type: Number,
+  },
+  trackName: {
+    type: String,
+  },
+  artistName: {
+    type: String,
+  },
+  genre: {
+    type: String,
+  },
+  popularity: {
+    type: Number,
+  }
```

Comment:
It will be safer for you if you add `required: true` to some fields. this will prevent from creating empty entries in the database ;)